scripts/jtl_listener_service.py

```diff
@@ -50,19 +57,46 @@ def __init__(
         events.test_start.add_listener(self._test_start)
         events.test_stop.add_listener(self._test_stop)
 
+    def _enqueue_results(self, results, cpu_usage):
+        self.queue.put_nowait({
+                "itemId": self.item_id,
+                "samples": results,
+                "monitor": cpu_usage,
+            })
+        
+    def _dequeue_results(self, name):
+        while True:
+            gevent.sleep(self.dequeuers_config["send_interval"])
+            try:
+                self._log_results(payload=self.queue.get(timeout=10))
+            except:
+                if len(self._background_dequeuers)>1 or self._finished:
+                    logging.info(f"Stoping jtl-reporter {name}")
+                    break
+
+    def _update_dequeuers_threads_quantity(self):
+        while not self._finished:
```

Comment:
I dont have experience with Queue module, but do we really need the while loop to start the dequeuers?